proto/substrait/algebra.proto

```diff
@@ -225,6 +225,9 @@ message AggregateRel {
     // within the measure.
     // Helps to support SUM(<c>) FILTER(WHERE...) syntax without masking opportunities for optimization
     Expression filter = 2;
+    // An optional sort expression that acts to define required ordering for the aggregator.
```

Comment:
Required ordering?  Does that mean the order that the aggregator expects its input to have or how it is expected to provide its output?